README.md

```diff
@@ -113,8 +113,27 @@ Key Transparency and its [Trillian](https://github.com/google/trillian) backend
 use a [MySQL database](https://github.com/google/trillian/blob/master/README.md#mysql-setup),
 which must be setup in order for the Key Transparency tests to work.
 
-Support
-------
+
+### Directory structure
+
+The directory structure of Key Transparency is as follows:
+
+* [**core**](core): main library source code. Core libraries do not import [impl](impl).
+    * [**api**](core/api): gRPC API definitions
+    * [**keyserver**](core/keyserver): keyserver implementation
+    * [**sequencer**](core/sequencer): sequencer implementation
+    * [**adminserver**](core/adminserver): admin server implementation
+* [**impl**](impl): environment specific modules:
+    * [**authentication**](impl/authentication): grpc authentication interceptor
+    * [**sql**](impl/sql): mysql implementations of storage modules
+    * [**integration**](impl/integration): environment specific integration tests
+* [**cmd**](cmd): binaries
+    * [**keytransparency-client**](cmd/keytransparency-client): Key Transparency CLI client
+* [**scripts**](scripts): scripts
+    * [**deploy**](scripts/deploy.sh): Deploy to Google Compute Engine
```

Comment:
It might be worth ordering these in the same order as they are listed in the repo, for ease of navigation/lookup of what's in the repo.  But you may want to keep this ordering that lists them in order of importance (I'm assuming), so this is completely up to you :)

Reply:
It was in order of importance, maybe I can use **bold** to hint at what's important